<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDetails</class>
 <widget class="QDialog" name="SearchDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Engine</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(&quot;C:/Users/Tech Guy/Documents/QT Projects/GTSystem/img/superveloche.png&quot;);
color:rgb(255, 255, 255);
</string>
  </property>
  <widget class="QLineEdit" name="txtsearch">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Enter Registration_No/Course_ID/Program_ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:rgb(255, 255, 255);
border-style: outset;
 border-width: 3px;
    border-radius: 20px;
    border-color:rgb(37, 197, 197);
  /*  min-width: 10em;*/
    padding: 6px;</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>20</width>
     <height>381</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="sfirstname">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Update First Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sregno">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Update Registration_No</string>
   </property>
  </widget>
  <widget class="QLabel" name="slabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Choose Date Of Birth(MM/DD/YY)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateofbirth">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Update Date Of Birth</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Student Personal Details</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Course Detaills</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>380</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Program Details</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scourse3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Update Course Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scourse4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Update Course Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scourse6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Update Course Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scourse1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Update Course Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scourseid5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Course ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scourseid1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Course ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scourseid2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Course ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scourseid3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>210</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Course ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scourse2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Update Course Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scourseid4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Course ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scourseid6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Course ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scourse5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Update Course Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sprogramid">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>420</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Update Program ID</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sprogram">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>Choose Program </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bsc.Applied Computer Science</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bsc.Computer Science</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-style: outset;
 border-width: 3px;
    border-radius: 20px;
    border-color:rgb(115, 210, 22);
  /*  min-width: 10em;*/
    padding: 6px;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sdelete">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

border-style: outset;
 border-width: 3px;
    border-radius: 20px;
    border-color:rgb(204, 0, 0);
  /*  min-width: 10em;*/
    padding: 6px;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="slogout">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>460</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
 border-width: 3px;
    border-radius: 20px;
    border-color:rgb(206, 92, 0);
  /*  min-width: 10em;*/
    padding: 6p</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="slecture6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>330</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Lecturer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="slecture1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Lecturer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="slecture5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>290</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Lecturer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="slecture4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Lecturer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="slecture3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Lecturer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="slecture2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Lecturer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:30px;
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>GT_SEARCH ENGINE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="udrecord">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Update/Delete Record</string>
   </property>
  </widget>
  <widget class="QLabel" name="slabel_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Update/Delete Records</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="camaroprogram">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>420</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 189, 182);</string>
   </property>
   <property name="placeholderText">
    <string>Update Program Name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
